docs/merged_objects.md

```diff
@@ -134,7 +126,9 @@ Each such list will contain the following fields:
 
 | Item name                      | Contents                                                                                                                                                                                                                                                                                                                                                                           |
 | ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `salmon_version`               | Version of `salmon` used for initial mapping                                                                                                                                                                                                                                                                                                                                       |
+| `sample_id`                    | Sample ID in the form `SCPCS000000`                                                                                                                                                                                                                                                                                                                                                |
+| `library_id`                   | Library ID in the form `SCPCL000000`                                                                                                                                                                                                                                                                                                                                               || `salmon_version`               | Version of `salmon` used for initial mapping
+| `project_id`                   | Project ID in the form `SCPCP000000`                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
```suggestion
| `library_id`                   | Library ID in the form `SCPCL000000`                                                                                                                                                                                                                                                                                                                                               |
| `project_id`                   | Project ID in the form `SCPCP000000`                                                                                                                                                                                                                                                                                                                                               |
| `salmon_version`               | Version of `salmon` used for initial mapping
|
```
I feel like this order should be switched, even if it's not this order in the object this makes more sense to me and order shouldn't matter in the actual object.